gapipy/request.py

```diff
@@ -95,7 +95,11 @@ def get(self, resource_id=None, uri=None, variation_id=None):
             raise ValueError(
                 'Need to provide at least one of `resource_id` or `uri` as argument')
         if not uri:
-            components = ['', self.resource, str(resource_id)]
+            uri_override = self._get_uri_override()
+            if uri_override:
```

Comment:
Similar to my comment re. id, looks like we can just set the `uri` on the base model to be the defined `self.resource`

Then, if we want to override like you need to in Token, you can just set `uri = 'token'`

I prefer this approach mostly for the sake of introspection of a gapipy Resource instance. A developer can safely assume a `self.uri` attribute will exist on the object with the useful information it contains.